synchronization/src/androidTestCyface/java/de/cyface/synchronization/MockedHttpConnection.java

```diff
@@ -44,4 +46,9 @@ public HttpURLConnection openHttpConnection(@NonNull URL url) throws ServerUnava
             throws ResponseParsingException, UnauthorizedException {
         return new HttpResponse(201, "");
     }
```

Comment:
we now have a second post() function for the binary upload (which uses slightly different options than the auth post request). Things will be cleaned up with the upcoming milestone - the file uploader.